src/main/java/at/fhv/teama/easyticket/client/jfx/views/veranstaltung/VeranstaltungController.java

```diff
@@ -117,7 +117,7 @@
     private LocalDate _filterDatumTo;
     private String _filterGenre;
 
-    private Model model = Model.getInstance();
+    private final Model model;
 
     //region Handlers
     private final EventHandler<ActionEvent> onFilterChanged = new EventHandler<ActionEvent>() {
```

Comment:
Der Eventhandler ist wie eine innere Klasse du hast von dort aus zugriff auf die Attribute in der umschliessenden Klasse. Daher kannst du hier einfach das Model von aussen verwenden.